web/src/utils/submitListUtils.ts

```
@@ -10,7 +10,7 @@ import { registrationTemplate, removalTemplate, dataMappings } from "@kleros/cur
 export const constructListParams = (listData: IListData, listMetadata: IListMetadata) => {
   const baseTemplate = { ...listData } as IList;
 
-  if (!isUndefined(listMetadata.policyURI) && listMetadata.policyURI === "") delete listMetadata.policyURI;
+  if (!isUndefined(listMetadata.policyURI) && isEmpty(listMetadata.policyURI)) delete listMetadata.policyURI;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)